### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by mnibecker**
@go-oleg 
cc @demmer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,73 @@
 # Juttle Client Library
 
+Juttle Client Library is a set of lower level frontend building blocks for
```
Comment (go-oleg) [inline README.md]: `lower level` is relative, but not clear to what. Suggest removing and saying `set of frontend...`.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,73 @@
...
+## Running Example
 
-[![Juttle Ecosystem](https://github.com/juttle/juttle/raw/master/docs/images/JuttleEcosystemDiagram.png)](https://github.com/juttle/juttle/blob/master/docs/juttle_ecosystem.md)
+Here's how to run this library locally from this repo (also used for
+    development purposes):
 
+- Have a locally running instance of Juttle Service (or
+[Juttle Engine](https://github.com/juttle/juttle-engine)) on port 8080 (you
+can adjust the location by changing line 6 of `example/index.js`).
+- clone this library
```
Comment (go-oleg) [inline README.md]: I would write these as shell commands
```bash
$ git clone ...
$ npm install
$ cd example
$ ./server.js
```

**Comment by go-oleg** [comment]
lgtm

**Comment by mnibecker** [comment]
@go-oleg done and done

**Pull request merged**